### ISSUE TITLE
Errors vs warnings

### ISSUE THREAD

**Issue by max-mykhailenko** [roadhump/SublimeLinter-eslint#27]
All rules implements as errors, but script can work with these errors. Maybe linter should show yellow warning outline instead red?

**Comment by roadhump** [comment]
Check your .eslintrc file to set correct level of eslint awareness for each rule. It is not an issue of plugin.

**Comment by max-mykhailenko** [comment]
@roadhump Can't find a docs about that, can you add an example here?

**Comment by max-mykhailenko** [comment]
Example:
```
  "rules": {
    "quotes": [2, "single"],  // warning
    "no-multi-spaces" : true, // warning
    "key-spacing" : true, // warning
    "no-underscore-dangle" : false,
    "no-constant-condition" : true  // error
  }
```

**Comment by roadhump** [comment]
At the bottom of http://eslint.org/docs/configuring/
